### PR TITLE
SLES support

### PR DESCRIPTION
**Pull request by tuxmea**
#### Pull Request (PR) description
Add support for SLES via archive

#### This Pull Request (PR) fixes the following issues
n/a

### CODE REVIEW

**Review by dhoppe — APPROVED** [review]

File: files/telegraf.service [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[Unit]
```
Comment (bastelfreak) [inline files/telegraf.service:2]: can you add a header like `#THIS FILE IS MANAGED BY PUPPET`?

**Review by alexjfisher — COMMENTED** [review]
The module has some acceptance tests.  Perhaps we could turn them on for SUSE too now?

**Comment by bastelfreak** [comment]
at least back in the days, there were no suitable docker images for suse/sles so we abandoned the idea. I don't know if that works now. I would already be happy with unit tests.

**Comment by alexjfisher** [comment]
> at least back in the days, there were no suitable docker images for suse/sles so we abandoned the idea. I don't know if that works now. I would already be happy with unit tests.

oh, that's a shame.  I've no idea how compatible opensuse images would be or if there's even a version 12 one available.

**Review by alexjfisher — APPROVED** [review]

**Pull request merged**